extensions/azure/blob/build.gradle.kts

```diff
@@ -26,8 +26,8 @@ dependencies {
 
 publishing {
     publications {
-        create<MavenPublication>("azure.blob") {
-            artifactId = "azure.blob"
+        create<MavenPublication>("blob") {
```

Comment:
I know that `blob` is a specific azure service, but it's also a generic term widely used to describe a chunk of binary data. Maybe we should add the specification for this.

Reply:
fair point. maybe `blobstore` is better?

Reply:
maybe `blobstorage` because the official name is `Azure Blob Storage`